docs/pages/enterprise/sso.mdx

```diff
@@ -237,5 +237,88 @@ must be able to:
 If something is not working, we recommend to:
 
 - Double-check the host names, tokens and TCP ports in a connector definition.
-- Look into Teleport's audit log for claim mapping problems. It is usually stored on the
-  auth server in the `/var/lib/teleport/log` directory.ad
+- Review the below log information and the troubleshooting sections for the above guides.
+
+<Tabs>
+<TabItem label="Self-Hosted" scope={["oss","enterprise"]}>
+
+### Using the Web UI
+
+If you get "access denied" or other login errors, the number one place to check is the Audit
+Log on the Teleport Auth Server. You can access it in the **Activity** tab of the Teleport Web UI.
```

Comment:
```suggestion
Log. You can access it in the **Activity** tab of the Teleport Web UI.
```

Mentioning "on the Teleport auth server" is just confusing I think. In some cases the log is actually stored on the auth server, in others cases it is on external storage. Either way, the reader of this page doesn't care - their best place to go is always the web UI.